### PR TITLE
Added addons socials

### PR DESCRIPTION
**Pull request by TheRolfFR**
Made simple social icons,

image

Code is here, may need more style tuning in main.css

### CODE REVIEW

**Comment by RobertRR11** [comment]
Either I'm doing something very wrong or it doesn't seem to work?
image

**Comment by Juknum** [comment]
> Either I'm doing something very wrong or it doesn't seem to work? image

Can confirm, doesn't seems to be working on my side either

**Comment by RobertRR11** [comment]
> Either I'm doing something very wrong or it doesn't seem to work?

Also wanted to add that there are no relevant errors in console (only disqus errors because of locally hosting the site)

**Comment by RobertRR11** [comment]
Actually just noticed that it was because one of my uBlock Origin blocklists that removes social links. But there are still some issues:
1. Only shows up when there are multiple authors
2. Authors with no social links have an empty spot at the bottom
3. A massive gap when there are social links

image

File: js/addons/addon-post.js [hunk 1]
```diff
@@ -72,7 +72,18 @@ export default {
...
+                    <br>
+                    <div class="author-socials">
+                      <a v-for="m in author.media" :key="m.type + '-' + m.link" :href="m.link" target="_blank" rel="noreferrer" >
+                        <img v-if="MEDIAS_TO_ICONS[m.type].src" width="24" height="24" :src="MEDIAS_TO_ICONS[m.type].src" alt="m.type" />
+                        <i v-else-if="MEDIAS_TO_ICONS[m.type].fas" class="fas">{{ MEDIAS_TO_ICONS[m.type].fas }}</i>
+                        <i v-else-if="MEDIAS_TO_ICONS[m.type].fab" class="fab">{{ MEDIAS_TO_ICONS[m.type].fab }}</i>
+                        <i v-else class="fab">{{ MEDIAS_TO_ICONS["Other"].fab }}</i>
+                      </a>
+                    </div>
+                  </div>
```
Comment (Juknum) [inline js/addons/addon-post.js:86]: This somewhat never show inside the page 🤔

**Review by Juknum — CHANGES_REQUESTED** [review]

**Comment by Juknum** [comment]
> Actually just noticed that it was because one of my uBlock Origin blocklists that removes social links.
Is there a way to avoid uBlock Origin to block those links?

**Comment by RobertRR11** [comment]
> > Actually just noticed that it was because one of my uBlock Origin blocklists that removes social links.
> 
> Is there a way to avoid uBlock Origin blocking those links?

It's not being blocked by default, it's a special blocklist that needs to be activated manually.

**Comment by TheRolfFR** [comment]
So @RobertRR11 you shouldn't use the class for the icons because the class versions use the main css file from fontawesome. What I did in order to optimize speed on my website was getting rid of it and just imported the solid and brands styles with only the fonts. Much much lighter and reduces considerably the loading time.

**Comment by TheRolfFR** [comment]
Made it work with one author so you get this :
image

When I said icons may be tuning it was maybe make them all white in order to have a consistent look and alignment

**Comment by Juknum** [comment]
Seems good, but they aren't aligned on the y axis and this confuse me a lot xd

**Comment by RobertRR11** [comment]
Now everything looks good to me, but the last issue I see is that the paypal links for some reason also include the link of the hosted site
image

**Comment by RobertRR11** [comment]
> Now everything looks good to me, but the last issue I see is that the paypal links for some reason also include the link of the hosted site

Hey @TheRolfFR this is basically the last issue, can you look into it?

**Comment by TheRolfFR** [comment]
> Hey @TheRolfFR this is basically the last issue, can you look into it?

Just some missing http on the urls, simple fix on 27bd622
image
_Example at ``/addons/gilded-ores``_

**Review by RobertRR11 — APPROVED** [review]

**Pull request merged**